pkg/tests/integration/test_salt_ufw.py

```diff
@@ -0,0 +1,41 @@
+import os.path
+import pathlib
+import subprocess
+
+import pytest
+from pytestskipmarkers.utils import platform
```

Comment:
```suggestion
import pathlib

import pytest
```

Reply:
These are not used

Reply:
and pre-commit didn't mention that ?, will delete them

Reply:
No, it's pylint, and it's pretty relaxed right now, its config.

Reply:
done